specs/DisableNavigatingBackAndForward.md

```diff
@@ -0,0 +1,115 @@
+# API spec for disable navigating back/forward
+
+# Background
+This problem was first proposed by a developer on GitHub, who wants to prevent mouse Xbutton1 and XButton2. The reason for their superior level is to prevent users navigating back or forward.
```

Comment:
```suggestion
This problem was first proposed by a developer on GitHub, who wants to prevent users navigating back or forward using any of the built-in shortcut keys or special mouse buttons.
```

Reply:
update